java/dagger/hilt/android/processor/internal/androidentrypoint/AndroidEntryPointMetadata.java

```diff
@@ -271,7 +271,7 @@ private static AndroidEntryPointMetadata of(
           !MoreTypes.isTypeOf(Void.class, baseElement.asType()),
           androidEntryPointElement,
           "Expected @%s to have a value."
-          + " Did you forget to apply the Gradle Plugin?",
+          + " Did you forget to apply the Gradle Plugin? (dagger.hilt.android.plugin)",
```

Comment:
Should we add a link to the grade plugin doc? Otherwise I think just adding the id in the parentheses lacks context.

Reply:
makes sense - maybe the URL on a separate line, e.g. "See https://dagger.dev/hilt/gradle-setup.html"? Initially I wanted to put the whole `apply plugin: ''` line here, but Gradle has so many different ways to apply a plugin